DOC-1981: Added TINY-9632 to the release notes. (#2742)
tinymce/tinymce-docs: modules/ROOT/pages/6.5-release-notes.adoc

The following premium plugin updates were released alongside {productname} 6.5.

> === Spellchecker Pro 3.3.0
> 
> The {productname} 6.5 release includes an accompanying release of the **Spellchecker Pro** premium plugin.
> 
> **Spellchecker Pro** 3.3.0 includes one addition:
> 
> === Enhancements to Spellchecking Functionality in Spellchecker Pro
> //#TINY-9632
> 
> Spellchecker Pro introduces three new features with {productname} 6.5.
> 
> * Firstly, the addition of the `mceSpellcheckUpdate` command which allows users to automatically check their editor content for misspelled words. This command not only identifies the errors but also offers suggestions for correcting them.
> * Secondly, the `SpellcheckerUpdated` event, along with the accompanying spelling object, is triggered when the editor undergoes a spell check. This event is currently supported when using the `mceSpellcheckUpdate` command or when opening the dialog, but not in real-time as-you-type mode.
> * Lastly, the `getSpellingErrors()` API has been incorporated, enabling users to obtain an array that contains each misspelled word found in the content.
> 
> [NOTE]
> Read more about Spellchecker Pros new features xref:ROOT:introduction-to-tiny-spellchecker.adoc[here].
> 

Reviewer: ```suggestion
=== Spell Checker Pro 3.3.0

The {productname} 6.5 release includes an accompanying release of the **Spell Checker Pro** premium plugin.

**Spell Checker Pro** 3.3.0 includes three additions:

==== New `mceSpellcheckUpdate` command

The new `mceSpellcheckUpdate` command allows users to trigger a check for misspelled words in the editor content, and includes any spelling suggestions for the misspelt words. This allows users to verify their spelling all at once without the use of the Spellchecker dialog, with any suggestions available from a context menu.

==== New `SpellcheckerUpdated` event

The new `SpellcheckerUpdated` event is triggered each time the editor content is updated to display any new misspelt words, once suggestions are provided for those misspelt words. This event is currently supported when using the `mceSpellcheckUpdate` command or when opening the dialog. This event is currently not supported in As-you-type mode.

==== New `getSpellingErrors()` API
The new `getSpellingErrors()` API allows users to retrieve a list of each misspelling in the editor content.

[NOTE]
Read more about xref:ROOT:introduction-to-tiny-spellchecker.adoc[new Spell Checker Pro features here].

```
Not sure why we would say one addition, then detail three? Moving each under their own heading rather than enhancements. Also, the plugin name is Spell Checker Pro, I swear I always forget too.

The following premium plugin updates were released alongside {productname} 6.5.

=== Spell Checker Pro 3.3.0

The {productname} 6.5 release includes an accompanying release of the **Spell Checker Pro** premium plugin.

**Spell Checker Pro** 3.3.0 includes three additions:

==== New `mceSpellcheckUpdate` command

> The new `mceSpellcheckUpdate` command allows users to trigger a check for misspelled words in the editor content, and includes any spelling suggestions for the misspelt words. This allows users to verify their spelling all at once without the use of the Spellchecker dialog, with any suggestions available from a context menu.
> 
> ==== New `SpellcheckerUpdated` event
> 
> The new `SpellcheckerUpdated` event is triggered each time the editor content is updated to display any new misspelt words, once suggestions are provided for those misspelt words. This event is currently supported when using the `mceSpellcheckUpdate` command or when opening the dialog. This event is currently not supported in As-you-type mode.

Reviewer: ```suggestion
The new `mceSpellcheckUpdate` command allows users to trigger a check for misspelled words in the editor content, and includes any spelling suggestions for the misspelt words.

This allows users to verify their spelling all at once without the use of the Spellchecker dialog, with any suggestions available from a context menu.

==== New `SpellcheckerUpdated` event

The new `SpellcheckerUpdated` event is triggered each time the editor content is updated to display any new misspelt words, once suggestions are provided for those misspelt words.

This event is currently supported when using the `mceSpellcheckUpdate` command or when opening the dialog.

NOTE: This event is currently not supported in As-you-type mode.
```
Split up the paragraphs and modified into a note.

The following premium plugin updates were released alongside {productname} 6.5.

> === Spell Checker Pro 3.3.0
> 
> The {productname} 6.5 release includes an accompanying release of the **Spell Checker Pro** premium plugin.
> 
> **Spell Checker Pro** 3.3.0 includes three additions:
> 
> ==== New `mceSpellcheckUpdate` command
> 
> The new `mceSpellcheckUpdate` command allows users to trigger a check for misspelled words in the editor content, and includes any spelling suggestions for the misspelt words.
> 
> This allows users to verify their spelling all at once without the use of the Spellchecker dialog, with any suggestions available from a context menu.
> 
> ==== New `SpellcheckerUpdated` event
> 
> The new `SpellcheckerUpdated` event is triggered each time the editor content is updated to display any new misspelt words, once suggestions are provided for those misspelt words.
> 
> This event is currently supported when using the `mceSpellcheckUpdate` command or when opening the dialog.
> 
> NOTE: This event is currently not supported in As-you-type mode.
> 
> ==== New `getSpellingErrors()` API
> The new `getSpellingErrors()` API allows users to retrieve a list of each misspelling in the editor content.
> 
> [NOTE]
> Read more about xref:ROOT:introduction-to-tiny-spellchecker.adoc[new Spell Checker Pro features here].

Reviewer 2: ```suggestion
=== Spell Checker Pro 3.3.0

The {productname} 6.5 release includes an accompanying release of the **Spell Checker Pro** premium plugin.

**Spell Checker Pro** 3.3.0 includes three additions:

==== New `mceSpellcheckUpdate` command

The new `mceSpellcheckUpdate` command allows users to trigger a check for misspelled words in the editor content, and includes any spelling suggestions for the misspelt words.

This allows users to verify their spelling all at once without the use of the Spellchecker dialog, with any suggestions available from a context menu.

==== New `SpellcheckerUpdated` event

The new `SpellcheckerUpdated` event is triggered each time the editor content is updated to display new misspelt words once suggestions are provided for those misspelt words.

This event is currently supported when using the `mceSpellcheckUpdate` command or when opening the dialog.

NOTE: This event is not currently supported in As-you-type mode.

==== New `getSpellingErrors()` API

The new `getSpellingErrors()` API allows users to retrieve a list of each misspelling in the editor content.

For information on the **Spell Checker Pro** plugin, see: xref:introduction-to-tiny-spellchecker.adoc[Introduction to Spell Checker Pro].
```

1. Copy-edits.
2. Turned NOTE admonition into the standard _for information_ line.
